api/router.js

```diff
@@ -0,0 +1,17 @@
+const routes = {
+  '/auth/login': {
+    'handler': 'auth',
+    'action': 'login'
+  }
+}
+
+class Router {
+    static find(path) {
+      for (let route in routes) {
```

Comment:
I think you can get the value from the object directly by the key, and not use a slow loop for this.